buildSrc/subprojects/build-update-utils/src/main/kotlin/gradlebuild/buildutils/tasks/GenerateSubprojectsInfo.kt

```diff
@@ -42,6 +42,7 @@ abstract class GenerateSubprojectsInfo : DefaultTask() {
 
     private
     fun generateSubproject(subprojectDir: File): GradleSubproject {
+        // TODO (donat) we should probably change this one as well
         return GradleSubproject(subprojectDir.name,
             subprojectDir.name.kebabToCamel(),
```

Comment:
I think you need to remove the `kebabToCamel()` and regenerate the `subprojects.json` in order for CI to work correctly. You already adjusted the names in `lifecycle.gradle.kts`. The names are retrieved from the file that is generated by the code here I think.